scripts/rebase_patches/0020-klog-component-names.patch

```diff
@@ -0,0 +1,57 @@
+From 56e7ca38133a9f6c5edc3778e27308be9f8d397c Mon Sep 17 00:00:00 2001
+From: Ben Luddy <bluddy@redhat.com>
+Date: Wed, 31 Aug 2022 17:36:35 -0400
+Subject: [PATCH] Upsetting hack to tag klog with embedded component names.
```

Comment:
Would it be less upsetting to write a logr plugin that did this instead of using a patch to klog? I'm fine either way, but if it eases your conscience...

Reply:
I don't mind carrying the patch (given its size) as much the dependency on getProfLabel. I hope it will be a relatively stable interface for us given the age of the label feature, though.